fuzzers/Makefile

```diff
@@ -104,6 +104,7 @@ $(eval $(call fuzzer,056-pip-rem,049-int-imux-gfan 050-pip-seed 051-pip-imuxlout
 $(eval $(call fuzzer,057-pip-bi,056-pip-rem))
 ifneq ($(QUICK),Y)
 $(eval $(call fuzzer,058-pip-hclk,005-tilegrid))
+$(eval $(call fuzzer,059-pip-byp-bounce,048-int-piplist))
```

Comment:
pip-byp-bounce seems like it should be done before 056?

Reply:
Yes, you are correct. Fixed.